resources/definitions/fdmprinter.def.json

```diff
@@ -4336,6 +4333,39 @@
                         }
                     }
                 },
+                "support_interface_angles":
+                {
+                    "label": "Support Interface Line Directions",
+                    "description": "A list of integer line directions to use. Elements from the list are used sequentially as the layers progress and when the end of the list is reached, it starts at the beginning again. The list items are separated by commas and the whole list is contained in square brackets. Default is an empty list which means use the default angles(alternates between 45 and 135 degrees if interfaces are quite thick or 90 degrees).",
```

Comment:
Typo here: Needs a space before the bracket. I'll make this change as it's merged.

Reply:
Added a space before the bracket